pysqa/utils/core.py

```
@@ -0,0 +1,399 @@
+class CoreQueueAdapter(object):
+    def __init__(
+        self,
+        queue_type: str,
+        execute_command: callable = execute_command,
+    ):
+        self._commands = get_queue_commands(queue_type=queue_type)
+        if queue_type_dict[queue_type]["module_name"] is not None:
+            self._submission_template = getattr(
+                importlib.import_module(queue_type_dict[queue_type]["module_name"]),
+                "template",
+            )
```

Ruff

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)